src/modules/menu/components/search/SearchResultList.vue

```diff
@@ -1,31 +1,42 @@
 <template>
     <div
-        id="search-results"
-        class="bg-light border rounded overflow-hidden"
-        data-cy="search-results"
-        @keydown.esc.prevent="$emit('close')"
+        id="search-results-container"
+        :class="{ 'search-results-dev-site-warning': hasDevSiteWarning && isPhoneMode }"
     >
-        <SearchResultCategory
-            :title="$t('locations_results_header')"
-            :half-size="results.locationResults.length > 0"
-            :entries="results.locationResults"
-            data-cy="search-results-locations"
-            @preview-start="setPinnedLocation"
-            @preview-stop="clearPinnedLocation"
-        />
-        <SearchResultCategory
-            :title="$t('layers_results_header')"
-            :half-size="results.layerResults.length > 0"
-            :entries="results.layerResults"
-            data-cy="search-results-layers"
-            @preview-start="setPreviewLayer"
-            @preview-stop="clearPreviewLayer"
-        />
+        <div
+            class="search-results bg-light"
+            :class="
+                isPhoneMode
+                    ? ['border-top', 'border-bottom', 'rounded-bottom']
+                    : ['border', 'rounded']
+            "
+            data-cy="search-results"
+            @keydown.esc.prevent="$emit('close')"
+        >
+            <div class="search-results-inner">
+                <SearchResultCategory
+                    :title="$t('locations_results_header')"
+                    :half-size="results.locationResults.length > 0"
```

Comment:
we could get rid of this props binding as you've removed it from the SearchResultCategory component